xfer/publisher_test.go

```diff
@@ -1,16 +1,19 @@
-package xfer_test
+package xfer
```

Comment:
Pls don't change the package; @peterbourgon is very keen on having tests in separate packages, even if it means exposing slightly more of an API.